src/api/app/models/token/workflow.rb

```diff
@@ -13,68 +13,10 @@ def call(options)
 
     yaml_file = Workflows::YAMLDownloader.new(scm_webhook.payload, token: self).call
     workflows = Workflows::YAMLToWorkflowsService.new(yaml_file: yaml_file, scm_webhook: scm_webhook, token: self).call
-
-    case
-    when scm_webhook.closed_merged_pull_request?
-      destroy_target_projects(workflows)
-    when scm_webhook.reopened_pull_request?
-      restore_target_projects(workflows)
-    when scm_webhook.new_pull_request?, scm_webhook.updated_pull_request?
-      call_steps(workflows)
-    end
+    workflows.each { |workflow| workflow.call(user) }
```

Comment:
We don't need to pass the user to the workflow since the workflow already has an instance of the current token, so it's possible to do `token.user` in the workflow.

Reply:
True, fixed